browser/index.js

```diff
@@ -129,18 +123,6 @@ module.exports = (opts, userPlugins = []) => {
     : bugsnag
 }
 
-const getPrefixedConsole = () => {
```

Comment:
This is only available/used inside the `browser/index.js` exported function correct?
So it's removal shouldn't be a breaking change?

Reply:
Correct, it was "private" to that module. It's been moved to `browser/config.js` where it is now used.